### ISSUE TITLE
Translation Research

### ISSUE THREAD

**Issue by mvdicarlo** [mvdicarlo/postybirb#183]
Low priority task.

Translation is base supported in the app, but not much has been done with it.

Need to figure out how to export translation json for population.

Need to add a language dropdown (sidenav? settings?)

**Comment by leaftail1880** [comment]
#249

**Comment by leaftail1880** [comment]
just found out that new translation messages was not picked up by weblate because `lang/` symlink to `apps/postybirb-ui/lang/` was lost. Initially this was made for convenience (faster access to lang files, also we cannot move it outside of apps/postybirbi-ui/lang/ because of the vite dynamic import restrictions), but caused more problem, so i think that its better to completely remove `lang/` folder from the root instead of setting up shx cp script or symlink which may not work or cause problems

Will fix that after #272 merges, so no translation will be lost (spanish 100% already, huh)

**Comment by mvdicarlo** [comment]
Can maybe just have it part of the postinstall steps.

**Comment by leaftail1880** [comment]
@mvdicarlo i dont think thats the option because it may be unsynk and that may cause translation bugs we may not notice and postinstall does not runs before commit, so users will usually push unsynced versions. Maybe we should add beforecommit hook that will run lingui:extract and copy files to lang/?

**Comment by mvdicarlo** [comment]
Considering this done.
